### PR TITLE
Update clientutils.rst PR #1459

### PR DESCRIPTION
**Pull request by Macek007**
Doc update to PR n1k0#1459

### CODE REVIEW

File: docs/modules/clientutils.rst [hunk 1]
```diff
@@ -309,6 +309,43 @@ Logs a message with an optional level. Will format the message a way CasperJS wi
...
+``makeSelector()``
+-----------------------------------------------------------------------------
+
+**Signature:** ``makeSelector(String selector [, String type])``
+
+.. versionadded:: 1.1-beta5
+
+Makes selector by defined type XPath, Name or Label. Function has same result as selectXPath in Casper module for XPath type - it makes XPath object. Function also accepts name attribute of the form filed or can select element by its label text.
+
+Parametr ``type`` values:
```
Comment (mickaelandrieu) [inline docs/modules/clientutils.rst]: Parameter

File: docs/modules/clientutils.rst [hunk 2]
```diff
@@ -309,6 +309,43 @@ Logs a message with an optional level. Will format the message a way CasperJS wi
...
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+- 'css'
+   
+  CSS3 selector - selector is returned trasparently
```
Comment (mickaelandrieu) [inline docs/modules/clientutils.rst]: Transparently

File: docs/modules/clientutils.rst [hunk 3]
```diff
@@ -309,6 +309,43 @@ Logs a message with an optional level. Will format the message a way CasperJS wi
...
+   
+- 'xpath' || null
+    
+  XPath selector - return XPath object    
+
+- 'name' || 'names'
+ 
+  select input of specific name, internally covert to CSS3 selector
```
Comment (mickaelandrieu) [inline docs/modules/clientutils.rst]: converted into?

File: docs/modules/clientutils.rst [hunk 4]
```diff
@@ -360,6 +397,30 @@ Sends an AJAX request, using the following parameters:
...
 
+``setFieldValue()``
+-----------------------------------------------------------------------------
+
+**Signature:** ``setFieldValue(String|Object selector, Mixed value [, Object options])``
+
+.. versionadded:: 1.1-beta5
+
+Sets a value to form field by CSS3 or XPath selector.
+With `makeSelector()`_ function can by easily used with ``name`` or ``label`` selector
```
Comment (mickaelandrieu) [inline docs/modules/clientutils.rst]: can be

**Comment by BIGjuevos** [comment]
@mickaelandrieu @istr - This looks good to pull in to me.

**Comment by mickaelandrieu** [comment]
thanks @Macek007 and @BIGjuevos for review

**Pull request merged**